config/jacoco.gradle

```diff
@@ -29,39 +22,27 @@ task jacocoTestReport(type: JacocoReport) {
                          'android/**/*.*']
 
     def packagesExcluded = ['com/nimbl3/di/**',
-                            'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
     // def filesExcluded = [ ]
     // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
     def fileFilter = fileGenerated + packagesExcluded
 
-    classDirectories.from = fileTree(
-        dir: "$project.rootDir/app/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
-        excludes: fileFilter
-    ) + fileTree(
-        dir: "$project.rootDir/data/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+    classDirectories = fileTree(
+        dir: "$buildDir/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
```

Comment:
FYI this part will be changed to `$buildDir/intermediates/javac/stagingDebug/classes` after upgraded to 0.8.5 https://github.com/redplanethotels/rphl-android-app/pull/2109, I guess @jean-t86 and @Tuubz  know this.

Reply:
Let's not forget to check this when bringing back #41 @Wadeewee 🙏